index.html

```diff
@@ -33,6 +33,293 @@
+                    {{NotReadableError}} and abort these steps.
+                  </p>
+                  <p>If the result of the request is {{PermissionState/"granted"}} but device access fails for
+                    any reason other than those listed above, <a>reject</a>
+                    <var>p</var> with a new {{DOMException}}
+                    object whose {{DOMException/name}} attribute has the
+                    value {{AbortError}} and abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>Let <var>streams</var> be the
+                    list of {{MediaStream}} objects for which the permission was granted.</p>
+                </li>
+                <li>
+                  <p><a>Resolve</a> <var>p</var> with <var>streams</var> and
+                    abort these steps.</p>
+                </li>
+                <li>
+                  <p><em>Permission Failure</em>: [=Reject=]
+                    <var>p</var> with a new {{DOMException}}
+                    object whose {{DOMException/name}} attribute has the
+                    value {{NotAllowedError}}.
+                  </p>
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Return <var>p</var>.</p>
+            </li>
+          </ol>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST NOT capture content that's behind a
+            partially transparent captured <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>.
+          </p>
+          <p>
+            For the newly created {{MediaDevices/ScreenCaptureMediaStreamTrack}}s, the <a>user agent</a> MUST NOT capture the
+            prompt that was shown to the user.
+          </p>
+          <p>
+            Information that is not currently rendered to the screen SHOULD be obscured in captures
+            unless the application has been specifically authorized to access that content (e.g.
+            through means such as
+            <a data-cite="screen-capture#elevated-permissions">elevated permissions</a>).
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST NOT share audio.
+          </p>
+        </dd>
+      </dl>
+    </section>
+
+    <section>
+      <h2>ScreenCaptureMediaStreamTrack</h2>
+      {{MediaDevices/ScreenCaptureMediaStreamTrack}} extends {{MediaStreamTrack}} and
+      corresponds to a <a data-cite="screen-capture#dfn-monitor">monitor</a>.
+      It additionally provides a connection to the window placement API [[!window-placement]]
+      with the {{MediaDevices/ScreenCaptureMediaStreamTrack/screenDetailed}} function with which metadata about the
+      captured <a data-cite="screen-capture#dfn-monitor">monitor</a> can be retrieved.
+      <pre class="idl">
+        [Exposed=Window]
+        interface ScreenCaptureMediaStreamTrack : MediaStreamTrack {
+          ScreenDetailed screenDetailed();
```

Comment:
write a section about `screeDetailed` (maybe that's the reason why this is red?) and link to the window placement api algorithm (this may need an algorithm).